MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -356,6 +395,126 @@ namespace WPEFramework {
             LOGINFO("Worker Thread Completed");
         }
 
+        bool MaintenanceManager::knowWhoAmI()
+        {
+            bool success = false;
+            int retryDelay = 60;
+            const char* secMgr_callsign = "org.rdk.SecManager";
+            const char* secMgr_callsign_ver = "org.rdk.SecManager.1";
+            PluginHost::IShell::state state;
+            WPEFramework::JSONRPC::LinkType<WPEFramework::Core::JSON::IElement>* thunder_client = nullptr;
+
+            do {
+                if ((getServiceState(m_service, secMgr_callsign, state) == Core::ERROR_NONE) && (state == PluginHost::IShell::state::ACTIVATED)) {
+                    LOGINFO("%s is active", secMgr_callsign);
+
+                    thunder_client=getThunderPluginHandle(secMgr_callsign_ver);
+                    if (thunder_client == nullptr) {
+                        LOGINFO("Failed to get plugin handle");
+                        sleep(5);
+                    } else {
+                        JsonObject params;
+                        JsonObject joGetResult;
+
+                        params["clientId"] = "gdi";
+                        params["distributedTraceType"] = "money";
+                        params["distributedTraceId"] = "trace-id=8e44ec3c-7aa1-404d-a1e5-1dde72114ca3;parent-id=123;span-id=456";
```

Comment:
Can we manage these part of plugin configuration. Hardcoding is not a preferred way.